docs/blog/_posts/2018-10-05-10th-dotty-milestone-release.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.10.0-RC1
+author: Allan Renucci
+authorImg: /images/allan.jpg
+date: 2018-10-05
+---
+
+After a short summer break, we are excited to release Dotty version 0.10.0-RC1.
+This release serves as a technology preview that demonstrates new language features and the
+compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
+pioneered the fusion of object-oriented and functional programming in a typed setting. Scala 3 will
+be a big step towards realizing the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviors,
+- build on strong foundations to ensure the design hangs well together,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 10th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+## What’s new in the 0.10.0-RC1 technology preview?
+
+### Java 9+
+
+Dotty now supports the latest versions of Java. Try it out in our REPL!
+
+```scala
+scala> val ls = java.util.List.of(1, 2, 3)
+val ls: java.util.List[Int] = [1, 2, 3]
+```
+
+### Type-level programming: Match Types
+
+We introduces a new form of types called match types. A match type reduces to one of a number of
```

Comment:
introduces: typo. I also think that "A match type reduces ... scrutinee type" is a bit esoteric. Before this sentence, I would put a simpler definition as in the [pattern matching](https://docs.scala-lang.org/tour/pattern-matching.html).